src/components/Footer.tsx

```diff
@@ -1,5 +1,58 @@
+import FooterLogo from "../assets/FooterLogo.png";
+
 const Footer = () => {
-    return <>Footer</>;
+    return <>
+
+    <div className="flex flex-col sm:flex-row justify-between px-12 py-10 ">
+      {/* LEFT SIDE FOOTER */}
+      <div className="flex items-center">
+        <img draggable="false" src={FooterLogo} alt="ZK Email Footer Logo" />
+      </div>
+
+      {/* RIGHT SIDE FOOTER */}
+      <div className="flex flex-row justify-between gap-6 mt-[25px] sm:mt-[0px]">
+
+        <div className="flex flex-col gap-3">
+          <h4 className="bold">Developers</h4>
+          <a href='/' className="text-[#A8A8A8] font-400">Docs</a>
+          <a href='/' className="text-[#A8A8A8] font-400">Projects</a>
+        </div>
+
+        <div className="flex flex-col gap-3  ml-4">
+          <h4 className="bold">Developers</h4>
+          <a href='/' className="text-[#A8A8A8] font-400">Blogs</a>
+          <a href='/' className="text-[#A8A8A8] font-400 whitespace-nowrap">Learn More</a>
+          <a href='/' className="text-[#A8A8A8] font-400">Partner</a>
+        </div>
+
+        {/* ICON LINKS */}
+        <div className="flex flex-row gap-3 ml-4">
+          <a href='/https://x.com/zkemail?lang=en'>
+            <svg xmlns="http://www.w3.org/2000/svg" width="14" height="16" viewBox="0 0 14 16" fill="none">
```

Comment:
Do not add svgs directly into the files. It will clutter them with unnecessary things